content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,162 @@
+
+### Before starting
+
+Please review the [HPE GreenLake](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=GUID-D7192971-EF71-4304-B51E-548E7954E644.html) User Guide to understand how the SAML framework works in the context of Common Cloud Services for the HPE GreenLake edge-to-cloud platform for the HPE GreenLake Private Cloud Enterprise application .
+
+### Configure SSO/SAML applications in Okta
+
+   To configure application metadata in Okta, complete the following steps:
+
+* Step 1: Create an Okta SAML application
+* Step 2: Configure Sign On settings
+* Step 3: Export the SAML 2.0 IdP metadata
+* Step 4: Configure the SAML connection in the HPE GreenLake edge-to-cloud platform
+
+**Step 1: Create an Okta SAML application**
+
+1. Log into the Okta administration console.
+2. Click **Applications > Create new app integration.** The Create a new app integration window opens.
+3. Select SAML 2.0 and click **Next**.
+
+![](/img/ws-image0.png)
+
+Provide a name for the SAML application which gets connected to the HPE GreenLake edge-to-cloud platform
+
+![](/img/saml_app-okta.jpg)
+
+**Step 2: How to configure single sign-on settings**
+
+1. Enter the SAML information.
+
+   Under General:
+
+    **Single Sign on URL:** https://sso.common.cloud.hpe.com/sp/ACS.saml2
+
+    **Audience URI (SP Entity ID):** https://sso.common.cloud.hpe.com
+
+    **Name ID format EmailAddress**
+
+    **Application username Email**
+
+    **NameID = user.email**
+
+    **gl_first_name = user.FirstName**
+
+    **gl_last_name = user.LastName**
+
+    **hpe_ccs_attribute = (See Below)**
+
+   See here for IdP attribute details: [](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us)<https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=GUID-D7192971-EF71-4304-B51E-548E7954E644.html>
+
+   A new SAML attribute has been added “hpe_ccs_attribute” which tells HPE GreenLake edge-to-cloud platform and HPE GreenLake Private Cloud Enterprise application the exact role/permissions for each user. The following describes how to format the attribute.
+
+   Format: {version}#{pcid}:{app id}:{role_name}:{ALL_SCOPES}
+
+   N﻿ote :  At present HPE GreenLake Private Cloud Enterprise application role should be excluded.
+
+![](/img/hpe-greenlake-saml-attributes.jpg)
+
+![](/img/workspace-pcid.jpg)
+
+![](/img/glp_role_name.jpg)
+
+The **hpe_ccs_attribute** always starts with version1#. You must first configure the attributes for HPE GreenLake edge-to-cloud platform and To do so, enter the PCID for the account, followed by the HPE GreenLake application ID. This will always be **00000000-0000-0000-0000-000000000000**. Following this, enter the role name and ALL_SCOPES**.** 
```

Comment:
Make sure to escape special character (underscore) for the variables throughout your blog post:
The **hpe\_ccs\_attribute** always....